packages/builder/src/components/deploy/VersionModal.svelte

```diff
@@ -28,7 +30,9 @@
     $appStore.upgradableVersion &&
     $appStore.version &&
     $appStore.upgradableVersion !== $appStore.version
-  $: revertAvailable = $appStore.revertableVersion != null
+  $: revertAvailable =
```

Comment:
This is to allow setting the `revertableVersion` of older local apps that don't already have a `revertableVersion` set.